polaris-react/src/components/Page/Page.stories.tsx

```diff
@@ -346,3 +346,27 @@ export function WithContentAfterTitle() {
     </Page>
   );
 }
+
+export function WithContentAfterTitleAndSubtitle() {
+  return (
+    <Page
+      backAction={{content: 'Products', url: '#'}}
+      title="Jar With Lock-Lid"
+      titleMetadata={<Button size="large">Save</Button>}
```

Comment:
```suggestion
      titleMetadata={<Badge tone="attention">Verified</Badge>}
```

Using a button here feels like an odd pattern. Even if it is only for test purposes, can we use the Badge here similar to the other story?

Reply:
I used a button here to mimic orders as that is where this issue arose from.

![image](https://github.com/Shopify/polaris/assets/86790511/b53e4f20-07e5-4ef4-9222-a6d1c55ef150)

Issue with the badge is that I want this element to be 32px tall in order to demonstrate that it does not shift the header. We could expand the height of the badge, but then I think that also will look a bit funky.

Not sure what the most natural looking element would be here, but I could try to mimic the style of the Orders button for locations even closer if that's desired.

Reply:
Great point, thanks for the context.

Mimicking the "All locations 🔽 " would be great! Even if it isn't fully functional with the dropdown, I think having that would be ideal 👍 

Or even just using a large `tertiary`  button like this would be a nice closer match.

<img width="269" alt="17-14-ak398-yl36u" src="https://github.com/Shopify/polaris/assets/11774595/742691df-cb6e-4d76-ace0-f5e816b56572">

Reply:
Pushed!